sdk/core/abort-controller/review/abort-controller.api.md

```diff
@@ -21,10 +21,11 @@ export class AbortError extends Error {
 // @public
 export class AbortSignal implements AbortSignalLike {
     constructor();
-    readonly aborted: boolean;
+    get aborted(): boolean;
     addEventListener(_type: "abort", listener: (this: AbortSignalLike, ev: any) => any): void;
-    static readonly none: AbortSignal;
-    onabort?: (ev?: Event) => any;
+    dispatchEvent(_event: Event): boolean;
+    static get none(): AbortSignal;
+    onabort: ((ev?: Event) => any) | null;
```

Comment:
the only minor eyebrow raise here is the switch from onabort being optional to requiring `null`.

Looks like @daviwil did this 2 years ago though, so probably it's fine?